pkg/ebpf/telemetry/modifier.go

```diff
@@ -26,13 +26,33 @@ const (
 )
 
 // ErrorsTelemetryModifier is a modifier that sets up the manager to handle eBPF telemetry.
-type ErrorsTelemetryModifier struct{}
+type ErrorsTelemetryModifier struct {
+	trackedMaps []names.MapName
```

Comment:
can you add a short oneliner to explain the purpose of this field? 
like something with contextual info, and not just "used to track the maps"

Reply:
I have removed tracked maps altogther in favor of using MapSpecs https://github.com/DataDog/datadog-agent/pull/32650/commits/776db5cc1ac3f2cc6a657dfbe8c65bd7a3bf95e1